odxtools/decodestate.py

```diff
@@ -11,9 +11,9 @@ class ParameterValuePair(NamedTuple):
 
 class DecodeState(NamedTuple):
     """Utility class to be used while decoding a message."""
-    # bytes to be decoded
     coded_message: Union[bytes, bytearray]
-    # values of already decoded parameters
+    """bytes to be decoded"""
```

Comment:
these are doctrings for IDEs/linters, right? (i.e. python completely ignores them?)

Reply:
Yup, there is no `__doc__` or `help(...)` for attribute doc strings but, e.g., VSCode supports them and shows them on hover.